package.json

```diff
@@ -107,7 +107,7 @@
     "@types/source-map-support": "^0.5.6",
     "@types/uuid": "^9.0.0",
     "@types/yargs": "^17.0.22",
-    "better-sqlite3": "^8.1.0",
+    "better-sqlite3": "^11.8.1",
```

Comment:
Let's leave `package.json` unchanged for now, we still need `better-sqlite3@8` in `typeorm@0.3`, but we'll definitely bump it up to 11+ when we drop old NodeJs support (soon).

Currently we have `better-sqlite3@8.7.0` which uses SQLite `3.43.1`, that should be good enough for the tests (if you update the PR).